platform/web/js/libs/library_godot_audio.js

```diff
@@ -453,6 +460,18 @@ class SampleNode {
 		const bus = GodotAudio.Bus.getBus(params.busIndex);
 		const sampleNodeBus = this.getSampleNodeBus(bus);
 		sampleNodeBus.setVolume(options.volume);
+
+		(async () => {
+			/** @type {AudioWorkletProcessor | null} */
+			this._positionWorklet = await this.createPositionWorklet(true);
```

Comment:
I would add a `this.isCanceled` check inside the async function before doing anything. Async functions executed later after the main loop.